test/build/40_pull.bats

```diff
@@ -255,6 +255,8 @@ EOF
                   "$img"
     [[ $status -eq 0 ]]
     grep -F '"schemaVersion": 1' "${cache}/${img}.manifest.json"
+
+    run ch-image --storage="$unpack" delete 'debian:squeeze'
```

Comment:
I recommend changing this test to:

1. use a subdirectory of `$BATS_TMPDIR` for the storage directory, not `$BATS_TMPDIR` itself
2. remove `--no-cache`
3. just delete the whole storage directory with `rm -Rf`

With those changes, the following will be moot, but: Note also that if you wrap a command with `run`, you need to check the exit value and print the output manually. Here I'd suggest just omitting `run`.